recipe/meta.yaml

```
@@ -38,7 +44,7 @@ requirements:
     - swig                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}  # [linux64 or ppc64le or win]
+    - {{ compiler('cuda') }}  # [linux or win]
```

To ensure the CUDA compiler is only added when used, would suggest this tweak to the selector

```suggestion
    - {{ compiler('cuda') }}  # [(linux or win) and cuda_compiler != "None"]
```